internal/provider/resources/block.go

```diff
@@ -127,9 +129,64 @@ func (r *BlockResource) Create(ctx context.Context, req resource.CreateRequest,
 }
 
 // Read refreshes the Terraform state with the latest data.
-//
-//nolint:revive // TODO: remove this comment when method is implemented
 func (r *BlockResource) Read(ctx context.Context, req resource.ReadRequest, resp *resource.ReadResponse) {
+	var state BlockResourceModel
```

Comment:
Sometimes we call this `state`, sometimes `model`. The Terraform tutorial calls it `state`, which felt a little more intuitive for me when reading the rest of the function. But I'm good either way.

Reply:
yeah, i think we should eventually do away with the `model` naming.  IMO we should only have `state` or `plan` variables, based on which framework function (eg. `req.State.Get()` or `req.Plan.Get()`) is doing the unmarshalling

Reply:
Agreed 👍🏼 Opened https://github.com/PrefectHQ/terraform-provider-prefect/issues/195 so we can come back to that at some point.